mifosng-android/src/main/res/layout/fragment_generate_collection_sheet.xml

```diff
@@ -73,7 +73,7 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
-                android:text="14 July 2014"
+                android:text="@string/fragment_generate_collection_sheet_date"
```

Comment:
This is a placeholder. 

1. It should be used as ```tools:text```
2. No need to create a String resource for this.